plugin/src/main/java/net/mcjukebox/plugin/bukkit/MCJukebox.java

```diff
@@ -29,11 +31,21 @@
     @Getter private ShowManager showManager;
     @Getter private TimeUtils timeUtils;
 
+
+    private BukkitAudiences adventure;
+
+    public @NonNull BukkitAudiences adventure() {
+        if(this.adventure == null) {
+            throw new IllegalStateException("Tried to access Adventure when the plugin was disabled!");
```

Comment:
nitpick, but the if statement style should match the rest of the project (space after `if`)